static/app/utils/analytics/issueAnalyticsEvents.tsx

```diff
@@ -39,13 +39,15 @@ export type IssueEventParameters = {
   'issue_group_details.tags.bar.clicked': {
     is_mobile: boolean;
     tag: string;
+    type: string;
     value: string;
     platform?: string;
   };
   'issue_group_details.tags.show_all_tags.clicked': {
     is_mobile: boolean;
-    tag: string;
+    type: string;
```

Comment:
Maybe we can type this as `'bars' | 'distributions'` to avoid typos, or export the TagFacetsStyles type and use that here